.github/workflows/test_pr.yml

```diff
@@ -133,10 +145,11 @@ jobs:
       - name: Setup environment
         run: make setup
 
-      - name: Download Organization
+      - name: Test Raven
```

Comment:
Please add comment that we only test functionality of raven in different python versions based the exit code